packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -381,19 +403,91 @@ contract Reserve is
    * @return Returns true if the transaction succeeds.
    */
   function transferGold(address payable to, uint256 value) external returns (bool) {
+    address goldTokenAddress = registry.getAddressForOrDie(GOLD_TOKEN_REGISTRY_ID);
     require(isSpender[msg.sender], "sender not allowed to transfer Reserve funds");
     require(isOtherReserveAddress[to], "can only transfer to other reserve address");
     uint256 currentDay = now / 1 days;
     if (currentDay > lastSpendingDay) {
       uint256 balance = getUnfrozenReserveGoldBalance();
       lastSpendingDay = currentDay;
-      spendingLimit = spendingRatio.multiply(FixidityLib.newFixed(balance)).fromFixed();
+      spendingLimit = tokenDailySpendingRatio[goldTokenAddress]
```

Comment:
have you checked if `spendingLimit` is really needed?

Reply:
line 454